dev.lizardbyte.app.sunshine.yml

```diff
@@ -0,0 +1,215 @@
+        sha256: 9edf2006cc86a5055279647c38ec923d11a821ee4dc2c3033e8d20e8ee237cd9
+        x-checker-data:
+          type: anitya
+          project-id: 20540
+          stable-only: true
+          url-template: https://www.freedesktop.org/software/libevdev/libevdev-$version.tar.xz
+
+  - name: numactl
+    disabled: false
+    buildsystem: autotools
+    make-args:
+      - install
+    cleanup:
+      - "/bin"
+    sources:
+      - type: archive
+        url: https://github.com/numactl/numactl/archive/refs/tags/v2.0.16.tar.gz
+        sha256: a35c3bdb3efab5c65927e0de5703227760b1101f5e27ab741d8f32b3d5f0a44c
+        x-checker-data:
+          type: anitya
+          project-id: 2507
+          stable-only: true
+          url-template: https://github.com/numactl/numactl/archive/refs/tags/v$version.tar.gz
+
+  - name: intel-mediasdk
+    disabled: false
+    only-arches:
+      - x86_64
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DBUILD_SAMPLES=OFF
+      - -DBUILD_TUTORIALS=OFF
+      - -DENABLE_X11=OFF
+    sources:
+      - type: archive
+        url: https://github.com/Intel-Media-SDK/MediaSDK/archive/refs/tags/intel-mediasdk-23.2.0.tar.gz
+        sha256: 6b33ec917ef0c6395fb66c7b93aae4ae31b3d1dedb5d7d39e0c69f45acb7c4fb
+        x-checker-data:
+          type: anitya
+          project-id: 21814
+          stable-only: true
+          url-template: https://github.com/Intel-Media-SDK/MediaSDK/archive/refs/tags/intel-mediasdk-$version.tar.gz
+
+  - name: cuda
+    disabled: false
+    buildsystem: simple
+    only-arches:
+      - x86_64
+      - aarch64
+    cleanup:
+      - "*"
+    build-commands:
+      - chmod u+x ./cuda.run
+      - ./cuda.run --silent --toolkit --toolkitpath=$FLATPAK_DEST/cuda --no-opengl-libs --no-man-page --no-drm
+        --tmpdir=$FLATPAK_BUILDER_BUILDDIR
+      - rm -r $FLATPAK_DEST/cuda/nsight-systems-*
+      - rm ./cuda.run
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://developer.download.nvidia.com/compute/cuda/12.1.1/local_installers/cuda_12.1.1_530.30.02_linux.run
+        sha256: d74022d41d80105319dfa21beea39b77a5b9919539c0487a05caaf2446d6a70e
+        dest-filename: cuda.run
+      - type: file
+        only-arches:
+          - aarch64
+        url: https://developer.download.nvidia.com/compute/cuda/12.1.1/local_installers/cuda_12.1.1_530.30.02_linux_sbsa.run
+        sha256: 45ea4cd860f0a26d3db8ce032530f2ee0b55abdd587545213d395a73623b4278
+        dest-filename: cuda.run
+
+  - name: sunshine
+    disabled: false
+    buildsystem: cmake
+    builddir: true
+    build-options:
+      append-path: /usr/lib/sdk/node18/bin
+      cxxflags: -I${C_INCLUDE_PATH}/libevdev-1.0
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCMAKE_CUDA_COMPILER=/app/cuda/bin/nvcc
+      - -DSUNSHINE_ASSETS_DIR=share/sunshine
+      - -DSUNSHINE_EXECUTABLE_PATH="flatpak run dev.lizardbyte.sunshine"
```

Comment:
not sure what's this about.

Reply:
-DCMAKE_CUDA_COMPILER=/app/cuda/bin/nvcc is needed for finding the CUDA compiler

Custom CMake variables of the project
-DSUNSHINE_ASSETS_DIR=share/sunshine
-DSUNSHINE_EXECUTABLE_PATH="flatpak run dev.lizardbyte.sunshine"

Reply:
systemd? There is no systemd.

Reply:
I only care about line 182 here.

Reply:
`-DSUNSHINE_EXECUTABLE_PATH` is used in our source code. If this is how sunshine is launched when running the flatpak, then it is correct.

Reply:
and how does that work?

Reply:
Sorry, I was mistaken, currently it's only used in the service file. So however this is set is what is used when cmake configures the service file.

In the future we may use this value for other functions in the code... for example we are contemplating a method to restart our application on demand, so this variable would work perfectly for that.

Reply:
> In the future we may use this value for other functions in the code... for example we are contemplating a method to restart our application on demand, so this variable would work perfectly for that.

actually with this value it wouldn't work. To restart yourself in the sandbox, just exec yourself. It's not different.